README.md

```diff
@@ -1,9 +1,17 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/{APP-NAME}-app.svg?style=shield)](https://circleci.com/gh/giantswarm/{APP-NAME}-app)
+[![CircleCI](https://circleci.com/gh/giantswarm/{APP-NAME}-app.svg?style=shield)](https://circleci.com/gh/giantswarm/app-updater-app)
 
-# {APP-NAME}-app chart
+# app-updater-app chart
 
-Giant Swarm offers a {APP-NAME} Managed App which can be installed in tenant clusters.
-Here we define the {APP-NAME} chart with its templates and default configuration.
+Giant Swarm offers a App Updater which can be installed in clusters to update App Platform apps when there is a new version.
+
+It uses a simple bash script in a `ConfigMap` that pulls from Github the last release and compares with the version set in the App Custom Resource. If it is different then set the new version on the App CR. It runs by default every 30 minutes as `Cronjob` Kubernetes entity.
```

Comment:
It seems to me that there is no Semver comparison of the current and the new release. So it could be that 1.1.0 is deployed, then 1.0.1 gets released, and as a result 1.1.0 would be replaced by 1.0.1.

For our docs, I don't see a problem, but it might be worth noting here as a limitation.

Reply:
yeah I thought about it. But in my opinion, you should be able to downgrade, so I have just checked it is a correct version and it is different. I added it here

Reply:
Thanks!